core/common/src/main/java/alluxio/util/webui/UIMetric.java

```diff
@@ -0,0 +1,39 @@
+package alluxio.util.webui;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * The type Ui metric.
+ */
+public class UIMetric {
+  private long mValue;
+
+  /**
+   * Instantiates a new Ui metric.
+   *
+   * @param value the value
+   */
+  @JsonCreator
+  public UIMetric(@JsonProperty("value") long value) {
```

Comment:
Isn't this just a wrapper around a long?

Reply:
I tried to remove it in favor of Long, but Jackson had issues, I may try again later.